src/api/db/job.ts

```diff
@@ -102,8 +117,7 @@ export function resetJob(jobId: number): Promise<any> {
       log: ''
     };
 
-    new Job({ id: jobId }).save(data, { method: 'update', require: false }).then(() => {
-      getJob(jobId).then(job => resolve(job));
-    });
+    new Job({ id: jobId }).save(data, { method: 'update', require: false })
+      .then(job => resolve(job.toJSON()));
   });
 }
```

Comment:
i am not sure, maybe my mistake, but check line `104` and `105`. this is uneccesarry additional database query. please remove it.